pubnub-core/src/subscription/subscribe_loop.rs

```diff
@@ -161,6 +161,7 @@ enum ControlOutcome {
 }
 
 /// Handle a control command.
```

Comment:
I wonder if we can get rid of this exception in the future by adding an async test or example.